### PR TITLE
Wiki Search Functionality

### PR DESCRIPTION
**Pull request by GeonheeLee**

### CODE REVIEW

**Comment by GeonheeLee** [comment]
Wrong branch

File: dli_app/mod_wiki/controllers.py [hunk 1]
```diff
@@ -139,3 +141,16 @@ def delete_page(page_id):
         db.session.commit()
 
     return redirect(url_for('wiki.home'))
+
+@mod_wiki.route('/search', methods=['POST', 'POST'])
```
Comment (gorel) [inline dli_app/mod_wiki/controllers.py:145]: Add one extra newline before this line (keep two lines between function routes)

File: dli_app/templates/wiki/view.html [hunk 2]
```diff
@@ -13,9 +13,14 @@
     </form>
   </div>
 
+ 
+
```
Comment (gorel) [inline dli_app/templates/wiki/view.html:17]: Remove these two extra lines

File: dli_app/templates/wiki/view.html [hunk 3]
```diff
@@ -13,9 +13,14 @@
...
   Table of Contents
   {{ toc|safe }}
   <hr>
 
   {{ html|safe }}
+
+  <a class="btn btn-danger" href="/wiki/home/">Go Back</a>
+
```
Comment (gorel) [inline dli_app/templates/wiki/view.html]: We don't need this link

**Pull request merged**